config/overlays/mutation_webhook/mutations.gatekeeper.sh_assign.yaml

```diff
@@ -0,0 +1,247 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.3.0
+  creationTimestamp: null
+  name: assign.mutations.gatekeeper.sh
+spec:
+  group: mutations.gatekeeper.sh
```

Comment:
rename to `mutation`?

Reply:
Can we address the mutations -> mutations change once everything is in?
I think it will be easier not to forget about pieces

Reply:
+1, This will require changes beyond the scope of this PR

Reply:
sgtm